docs/src/examples/hello-world.md

```diff
@@ -9,13 +9,13 @@ A "Hello World" type of program for the Fuel Indexer service.
 //! wasm32-unknown-unknown target will be required.
 //!
 //! ```bash
-//! cargo build -p hello-index --release --target wasm32-unknown-unknown
+//! cargo build -p hello_index --release --target wasm32-unknown-unknown
 //! ```
 //!
 //! Start a local test Fuel node
 //!
 //! ```bash
-//! cargo run --bin fuel-node
+//! fuel-core run --chain packages/fuel-indexer-tests/assets/test-chain-config.json
```

Comment:
@deekerno  
- So where is the contract deployment here? 
- The `fuel-node` that is being removed uses a deployed contract with a funded wallet that we can make contract calls against
- If we're gonna fully remove the `fuel-node` we need to update the example with these replacement steps
- I believe this just starts the node?